src/features/surveys/components/surveyForm/TextBlock.tsx

```diff
@@ -0,0 +1,8 @@
+import { FC } from 'react';
+import { ZetkinSurveyTextElement } from 'utils/types/zetkin';
+
+const TextBlock: FC<{ element: ZetkinSurveyTextElement }> = ({ element }) => {
```

Comment:
A text block can contain both a header and a paragraph. So I think this needs to be extended to display the `element.text_block.header` (if not an empty string).

Reply:
Fixed in 64b1b1a75ad4adf3f0a1d5cdd41668240da5f817!

I tested this by adding a BS extra element in `getServerSideProps`.

| | |
|-|-|
| <img width="878" alt="Screenshot 2023-12-10 at 20 13 57" src="https://github.com/zetkin/app.zetkin.org/assets/566159/665ec366-0f2a-4eb3-9193-bc3d31d1312c"> |  <img width="1133" alt="Screenshot 2023-12-10 at 20 13 40" src="https://github.com/zetkin/app.zetkin.org/assets/566159/f2b296eb-2681-4bf9-a8ca-71830c23c0a6"> |

Haven't touched the styling aspect of this yet. Seems like the survey subheading styling might need to be looked at as a whole thing cos currently there's lots of duplicated `style={}` props being thrown around for these elsewhere and it might be best to fix it all simultaneously with one `<SurveySubheading />` component or something.